opentelemetry-otlp/src/exporter/http/logs.rs

```diff
@@ -3,13 +3,15 @@ use std::sync::Arc;
 use async_trait::async_trait;
 use http::{header::CONTENT_TYPE, Method};
 use opentelemetry::logs::{LogError, LogResult};
-use opentelemetry_sdk::export::logs::{LogData, LogExporter};
+use opentelemetry::InstrumentationLibrary;
+use opentelemetry_sdk::export::logs::LogExporter;
+use opentelemetry_sdk::logs::LogRecord;
 
 use super::OtlpHttpClient;
 
 #[async_trait]
 impl LogExporter for OtlpHttpClient {
-    async fn export<'a>(&mut self, batch: Vec<std::borrow::Cow<'a, LogData>>) -> LogResult<()> {
+    async fn export(&mut self, batch: Vec<(&LogRecord, &InstrumentationLibrary)>) -> LogResult<()> {
```

Comment:
Hmm can we wrap it with a struct, otherwise it will hard to add new field/data in the future

Reply:
I think we still have flexibility to add new fields in `LogRecord` structure, as that represents the actual Log datamodel.